tests/OpenTokTest/LayoutTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace OpenTokTest;
+
+use OpenTok\Layout;
+use PHPStan\Testing\TestCase;
+
+class LayoutTest extends TestCase
+{
+    public function testStylesheetIsNotInSerializedArrayIfNotCustom()
+    {
+        $layouts = [
+            Layout::LAYOUT_BESTFIT => Layout::getBestFit(),
+            Layout::LAYOUT_HORIZONTAL => Layout::getHorizontalPresentation(),
+            Layout::LAYOUT_PIP => Layout::getPIP(),
+            Layout::LAYOUT_VERTICAL => Layout::getVerticalPresentation(),
+        ];
+
+        foreach ($layouts as $type => $object) {
+            $this->assertSame(['type' => $type], $object->toArray());
+        }
+    }
+
+    public function testStylesheetIsInSerializedArrayIfCustom()
+    {
+        $layout = Layout::createCustom(['stylesheet' => 'foo']);
+
+        $this->assertSame(['type' => LAYOUT::LAYOUT_CUSTOM, 'stylesheet' => 'foo'], $layout->toArray());
+    }
+
+    public function testScreenshareTypeSerializesProperly()
+    {
+        $layout = Layout::getBestFit();
+        $layout->setScreenshareType(Layout::LAYOUT_PIP);
+
+        $expected = [
+            'type' => 'bestFit',
+            'screenshareType' => 'pip'
+        ];
+
+        $this->assertSame($expected, $layout->toArray());
+    }
+
+    public function testScreenshareTypeCannotBeSetToInvalidValue()
+    {
+        $this->expectException(\RuntimeException::class);
+        $this->expectExceptionMessage('Screenshare type must be of a valid layout type');
+
+        $layout = Layout::getBestFit();
+        $layout->setScreenshareType('bar');
+    }
+
+    public function testScreenshareTypeCannotBeSetOnCustomLayout()
```

Comment:
According to the specs, the `type` must be set to `bestFit` specifically when using the `screenshareType`. Are you raising an exception if the `type` is not set to `bestFit` and not only when it's set to a broader custom layout type?

Reply:
I had that check wrong. It should complain on any layout that isn't best fit, so fixed